cfme/fixtures/v2v.py

```diff
@@ -41,6 +37,292 @@ def conversion_tags(appliance, host_creds):
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
         display_name='VDDK')
+    host_creds.remove_tags(host_creds.get_tags())
     host_creds.add_tags(tags=(tag1, tag2))
     yield
     host_creds.remove_tags(tags=(tag1, tag2))
+
+
+def get_vm(request, appliance, second_provider, template, datastore='nfs'):
+    """Fixture to provide vm object"""
+    source_datastores_list = second_provider.data.get('datastores', [False])
+    if not source_datastores_list:
```

Comment:
I don't think this is what you want, this won't skip on the default of `[None]`:
```
In [6]: if not [None]:
   ...:     print('skipping')
   ...:     

In [7]: if not []:
   ...:     print('skipping')
   ...:     
skipping
```

You want to use a default empty list for `source_datastores_list`, or `False`, but not both.